vscode-extensions/tls-sync-vscode-workspace-extension/package.json

```diff
@@ -0,0 +1,151 @@
+{
+  "name": "tls-sync-vscode-workspace-extension",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/rushstack.git",
+    "directory": "vscode-extensions/tls-sync-vscode-workspace-extension"
+  },
+  "license": "MIT",
+  "publisher": "RushStack",
+  "preview": true,
+  "displayName": "TLS Sync (Workspace Extension)",
+  "description": "",
+  "homepage": "https://rushstack.io",
+  "extensionKind": [
+    "workspace"
+  ],
+  "categories": [
+    "Other"
+  ],
+  "keywords": [],
+  "galleryBanner": {
+    "color": "#f0f0f0",
+    "theme": "light"
+  },
+  "engines": {
+    "vscode": "^1.98.0"
+  },
+  "main": "./extension.js",
+  "scripts": {
+    "build": "heft build --clean",
+    "build:watch": "heft build-watch",
+    "start": "heft start",
+    "pretest": "npm run build",
+    "test": "node ./lib-commonjs/test/runTest.js",
+    "_phase:build": "heft build --clean",
+    "_phase:test": ""
+  },
+  "contributes": {
+    "commands": [
+      {
+        "command": "tlssync.workspace.showLog",
+        "title": "Show Log",
+        "category": "TLS Sync"
+      },
+      {
+        "command": "tlssync.workspace.sync",
+        "title": "Sync TLS Certificates",
+        "category": "TLS Sync"
+      },
+      {
+        "command": "tlssync.workspace.showWalkthrough",
+        "title": "Show Walkthrough",
+        "category": "TLS Sync"
+      },
+      {
+        "command": "tlssync.workspace.showSettings",
+        "title": "Show Settings",
+        "category": "TLS Sync"
+      }
+    ],
+    "walkthroughs": [
+      {
+        "id": "sync-certificates",
+        "title": "TLS Sync",
+        "description": "Sync your debug TLS certificates with the remote machine.",
+        "featuredFor": [
+          ".tlssync"
+        ],
+        "steps": [
+          {
+            "id": "runsynccommand",
+            "title": "Sync Certificates",
+            "description": "[Sync](command:tlssync.workspace.sync)",
+            "completionEvents": [
+              "onContext:tlssync.workspace.sync.complete"
+            ],
+            "media": {
+              "markdown": "./media/walkthrough-sync.md"
+            }
+          },
+          {
+            "id": "changetlssyncsettings",
+            "title": "Change Sync Settings",
+            "description": "[Open Settings](command:tlssync.workspace.showSettings)",
+            "completionEvents": [],
+            "media": {
+              "markdown": "./media/walkthrough-settings.md"
+            }
+          }
+        ]
+      }
+    ],
+    "configuration": {
+      "title": "TLS Sync",
+      "type": "object",
+      "properties": {
+        "tlssync.storePath": {
+          "type": "string",
+          "title": "TLS Sync Store Path",
+          "description": "Directory where TLS certificates are read and written. The same directory is used for both the UI and workspace extensions."
```

Comment:
If the UI and workspace are on different operating systems, the same value doesn't make sense in both, unless this is specified as being relative to the user's home directory.

Reply:
I'll split the store path into `tlssync.local.storePath` and `tlssync.remote.storePath`